core/server/lib/image/manipulator.js

```diff
@@ -8,60 +8,21 @@ const fs = require('fs-extra');
  * We currently can't enable compression or having more config options, because of
  * https://github.com/lovell/sharp/issues/1360.
  */
-const process = (options = {}) => {
-    let sharp, img, originalData, originalSize;
-
-    try {
-        sharp = require('sharp');
-    } catch (err) {
-        return Promise.reject(new common.errors.InternalServerError({
-            message: 'Sharp wasn\'t installed',
-            code: 'SHARP_INSTALLATION',
-            err: err
-        }));
-    }
-
-    // @NOTE: workaround for Windows as libvips keeps a reference to the input file
-    //        which makes it impossible to fs.unlink() it on cleanup stage
-    sharp.cache(false);
```

Comment:
This is unnecessary as we handle file opening and closing - not libvips

Reply:
? this avoids that sharp caches the input file and keeps a reference to the file.

Reply:
Sharp doesn't have a reference to the file, it just works on raw data as of https://github.com/TryGhost/Ghost/commit/28a686a936653fefc466f0f93b1210703d26fb43#diff-58eacecfe1e5c6bf4e29d13e7893b571 - we no longer pass in the file name as the input - but the raw data :relaxed:

Reply:
Oh yeah, it was changed when we added the size comparison 👍 👻